gamla/functional.py

```diff
@@ -213,6 +213,10 @@ def wrap_tuple(x):
     return (x,)
 
 
+def wrap_dict(key: Text):
```

Comment:
consider `value_to_dict` (took me time to understand because the name is confusing)